src/components/ShowAllMonsters.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react'
+import { searchMonster } from '../actions/actions'
+import { connect } from 'react-redux'
+import './ShowAllMonsters.css'
+
```

Comment:
It will be interesting to look at maybe incorporating hooks here as well. :rubshandsmenacingly: